src/components/buttons/aprove-dev-request.js

```diff
@@ -23,5 +23,18 @@ module.exports = {
     await interaction.message.thread.setArchived(true);
 
     await interaction[interaction.replied || interaction.deferred ? 'followUp' : 'reply']({ content: "Desenvolvedor Aprovado", ephemeral: true });
+  
+    const webhook = parseWebhookURL(process.env.OFFTOPIC_WEBHOOK);
+    await interaction.client.fetchWebhook(webhook.id, webhook.token)
+        .then((webhook) => {
+            webhook.send({
+                content: `<:icons_djoin:875754472834469948> O membro **${member}** entrou como desenvolvedor na EPF`,
```

Comment:
`member` vai já ser parseado pra menção não precisa de markdown
```suggestion
                content: `<:icons_djoin:875754472834469948> O membro ${member} foi aprovado como desenvolvedor na EPF`,
```